### DISCUSSION TITLE
你好，请问simbot3.x目前支持发视频么？

### DISCUSSION THREAD

**Discussion by heixish** [ForteScarlet/simpler-robot#364] [不要在这里发帖子了]
<groupId>love.forte.simbot.boot</groupId>
<artifactId>simboot-core-spring-boot-starter</artifactId>

<groupId>love.forte.simbot.component</groupId>
<artifactId>simbot-component-mirai-boot</artifactId>

<simbot.version>3.0.0.preview.16.0</simbot.version>

**Comment by ForteScarlet** [comment]
simbot的 [mirai组件](https://github.com/simple-robot/simbot-component-mirai) 依赖于开源框架 [mirai](https://github.com/mamoe/mirai)。mirai目前并不支持视频发送，因此组件也不可以。
